gateways/cybersource/request_builders.go

```diff
@@ -92,6 +114,62 @@ func buildAuthRequest(authRequest *sleet.AuthorizationRequest) (*Request, error)
 			},
 		},
 	}
+
+	// Apple Pay request
+	if authRequest.Cryptogram != "" {
+		request.PaymentInformation = &PaymentInformation{
+			TokenizedCard: &TokenizedCard{
+				Number: authRequest.CreditCard.Number,
+				ExpirationYear: strconv.Itoa(authRequest.CreditCard.ExpirationYear),
+				ExpirationMonth: fmt.Sprintf("%02d", authRequest.CreditCard.ExpirationMonth),
+				TransactionType: "1",	// in-app transaction
```

Comment:
I think "in-app" is something like app related transaction like applepay, googlepay, samsungpay. But because I can make payment on browser, not sure is it good to call it mobile app?

Reply:
Yeah do they have a generic "ecommerce" transaction type or something similar that we can use instead?

Reply:
So from the doc (https://developer.cybersource.com/api/developer-guides/dita-payments/CreatingOnlineAuth/CreatingAuthReqPNT.html#id189DF0WI0HT_CreatingAuthReqPNT): it just mentioned
`Include the payment network transaction type field, transactionType, under the tokenizedCard object. The only supported value is 1, and it indicates that this is an in-app transaction.`

Also from API doc (https://developer.cybersource.com/api-reference-assets/index.html#payments_payments_process-a-payment_requestexpandedfielddescription_paymentInformation_tokenizedCard):
it explained
```transactionType
Type stringMaxLength 1
Type of transaction that provided the token data. This value does not specify the token service provider; it specifies the entity that
provided you with information about the token.

Possible value:

2: Near-field communication (NFC) transaction. The customer’s mobile device provided the token data for a contactless EMV transaction. For recurring
transactions, use this value if the original transaction was a contactless EMV transaction.
NOTE No CyberSource through VisaNet acquirers support EMV at this time.

Required field for PIN debit credit or PIN debit purchase transactions that use payment network tokens; otherwise, not used.
```
where is not very clear what option else I can use